### ISSUE TITLE
Custom model binder class not picked up in Model Binding tab.

### ISSUE THREAD

**Issue by MrClyfar** [Glimpse/Glimpse#322]
Hi.

I have the following view model class:

```
public class HomeIndexViewModel
{
    public string Name { get; set; }
    public int Age { get; set; }
}
```

Here is the custom model binder:

```
public class MyModelBinder : IModelBinder
{
    public object BindModel(ControllerContext controllerContext, ModelBindingContext bindingContext)
    {
        var viewModel = new HomeIndexViewModel();

        viewModel.Name = "From model binder.";
        return viewModel;
    }
}
```

and how I use that model binder:

```
[HttpPost]
public ActionResult Index([ModelBinder(typeof(MyModelBinder))]HomeIndexViewModel viewModel)
{
    return this.RedirectToAction("SaveSuccess");
}

public ActionResult SaveSuccess(HomeIndexViewModel viewModel)
{
    return this.View();
}
```

Now, when I have Glimpse open, and I view the POST request for "/Index" (via the History tab, and clicking "Inspect"), the "Model Binding" tab is not enabled.

If I remove the `[ModelBinder]` attribute, and instead register the model binder in `Global.asax.cs`

```
ModelBinders.Binders.Add(typeof(HomeIndexViewModel), new MyModelBinder());
```

and analyze the POST request again, this time the "Model Binding" tab is enabled.

Basically it looks like Glimpse is having trouble recognizing that my custom model binder was used during a request when using the `[ModelBinder]` attribute.

Thanks.
